shaders/vertex.glsl

```diff
@@ -1,9 +1,10 @@
-precision mediump float;
+precision highp float;
```

Comment:
I'm curious. Does bumping the precision to `highp` affect the performance? In other words, why would anyone use `mediump` to begin with?

Reply:
Good question.
Actually we could switch back to mediump in all shader programs. I didn't see much improvement as a results of this change. But the other changes was helpful.